### ISSUE TITLE
NullPointerException in javax.swing.ToolTipManager$insideTimerAction.actionPerformed

### ISSUE THREAD

**Issue by BlueThing00** [MegaMek/mekhq#682]
### Environment
MekHQ: 0.43.9-RC3
OS: Windows 10
Java: 1.8.0_161-b12

### Description
While looking for errors coming from Princess when she hangs, I saw a lot of NPEs from ToolTipManager in the log.

The callstack looks like this:
```
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
	at javax.swing.ToolTipManager$insideTimerAction.actionPerformed(Unknown Source)
	at javax.swing.Timer.fireActionPerformed(Unknown Source)
	at javax.swing.Timer$DoPostEvent.run(Unknown Source)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$500(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
```

### Files
[mekhqlog.bak.txt](https://github.com/MegaMek/mekhq/files/1857099/mekhqlog.bak.txt)

**Comment by NickAragua** [comment]
I believe this issue has been addressed with Megamek PR #945, so it should stop occurring as of the next release.

**Comment by BlueThing00** [comment]
Thank you for the prompt response.  Glad it's fixed.
